lxd/auth/oidc/oidc.go

```diff
@@ -371,6 +375,30 @@ func (*Verifier) setCookies(w http.ResponseWriter, idToken string, refreshToken
 	return nil
 }
 
+func (o *Verifier) secureCookieFromSession(sessionID uuid.UUID) (*securecookie.SecureCookie, error) {
```

Comment:
missing doc block explaining what this function does

Reply:
in general I think its prudent to add a lot more comments to this function explaining what each step does, and most importantly why. As well as a warning in the doc block that changing this implementation may cause existing users to be logged out.

Reply:
I've added a bunch of comments. Let me know if there is anything else worth adding or if it is unclear.